python/lsst/ap/association/ssoAssociation.py

```diff
@@ -163,8 +163,14 @@ def run(self, diaSourceCatalog, solarSystemObjects, exposure):
                 decs.append(dia_dec)
                 expected_ras.append(ssObject["ra"])
                 expected_decs.append(ssObject["dec"])
-
-        self.log.info("Successfully associated %d SolarSystemObjects.", nFound)
+                associated.append(True)
```

Comment:
`associated`  is only used to set `maskedObjects['associated']` below. Could you instead set the appropriate index of `maskedObjects['associated']` directly to `True` or `False` here?